### PR TITLE
Add backup logic to generate keys as part of the flow

### LINKED ISSUE #319: Add key generation support to Dapp bootstrap
### Problem

Currently the Dapp bootstrap flow expects keys.
In some cases it's easier to generate new keys as part of the flow

### Solution

Add backup logic to generate keys as part of the flow

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by Nana-EC**
Signed-off-by: Nana-EC <nana@swirldslabs.com>

**Description**:
Add backup logic to generate keys as part of the flow

**Related issue(s)**:

Fixes #319 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by ar-conmit — APPROVED** [review]
LGTM

File: dapp-example/tests/support/bootstrap.js [hunk 1]
```diff
@@ -28,11 +28,13 @@ if (supportedEnvs.includes(network.toLowerCase())) {
 }
 
 client.setOperator(process.env.OPERATOR_ID_MAIN, process.env.OPERATOR_KEY_MAIN);
+console.log(`SDK setup for ${JSON.stringify(network)} for account: ${process.env.OPERATOR_ID_MAIN}`);
 
 const createAccountFromCompressedPublicKey = async function(compressedPublicKey) {
   const transferTransaction = await (new HederaSDK.TransferTransaction()
-    .addHbarTransfer(HederaSDK.PublicKey.fromString(compressedPublicKey).toAccountId(0, 0), new HederaSDK.Hbar(100))
-    .addHbarTransfer(HederaSDK.AccountId.fromString(process.env.OPERATOR_ID_MAIN), new HederaSDK.Hbar(-100)))
+    .addHbarTransfer(HederaSDK.PublicKey.fromString(compressedPublicKey).toAccountId(0, 0), new HederaSDK.Hbar(10))
```
Comment (natanasow) [inline dapp-example/tests/support/bootstrap.js:35]: Why decrease the initial balance from 100 to 10 HBARs?

**Review by natanasow — APPROVED** [review]
LGTM

File: dapp-example/tests/support/bootstrap.js (hunk 1 above)
Comment (Nana-EC) [reply dapp-example/tests/support/bootstrap.js:35]: It's expensive when running against remote envs.
HBAR is cheap on local-node

**Pull request merged**